main.html

```diff
@@ -0,0 +1,333 @@
+                    style="width: 784px; height: 348px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+                <h1 id="Modal" style="color: #CC3300;">Modals</h1>
+                <div class="modals">
+                    <h1>This is modal</h1>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. A, delectus hic quidem excepturi ipsa nisi soluta animi asperiores! Non quo unde exercitationem voluptatem, dicta assumenda.
+                    </p>
+                    <button class="cross">x</button>
+                    <button class="outline-danger button">Close</button>
+                    <button class="outline-success button">Continue</button>  
+                </div>
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28155%2C155%2C155%2C1%29&t=cobalt&wt=sharp&l=auto&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522modals%2522%253E%250A%2509%253Ch1%253EThis%2520is%2520modal%253C%252Fh1%253E%250A%2520%2520%2520%2520%253Cp%253ELorem%2520ipsum%2520dolor%2520sit%2520amet%252C%2520consectetur%2520ad%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%253C%252Fp%253E%250A%2520%2520%2520%2520%253Cbutton%2520class%253D%2522cross%2522%253Ex%253C%252Fbutton%253E%250A%2520%2520%2520%2520%253Cbutton%2520class%253D%2522outline-danger%2520button%2522%253EClose%253C%252Fbutton%253E%250A%2520%2520%2520%2520%253Cbutton%2520class%253D%2522outline-success%2520button%2522%253EContinue%253C%252Fbutton%253E%2520%2520%250A%253C%252Fdiv%253E"
+                    style="width: 723px; height: 330px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+                <h1 id="Input" style="color: #CC3300;">Input</h1>
```

Comment:
give the classname as the color name you are providing instead of using inline styling.